src/main/kotlin/racingcar/InputValidator.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+object InputValidator {
+	private const val MAX_NAME_LENGTH = 5
+	private const val MIN_TRY_COUNT = 1
+
+	fun validateName(names: List<String>) {
+		names.forEach() { name ->
+			when {
+				name.isEmpty() -> throw IllegalArgumentException("참가자 이름은 비어있을 수 없습니다.")
+				name.length > MAX_NAME_LENGTH -> throw IllegalArgumentException("참가자 이름은 ${MAX_NAME_LENGTH}자 이하여야 합니다.")
```

Comment:
throw 대신 require를 쓰면 어떨까요??
```suggestion
				require(name.isNotEmpty()) { "참가자 이름은 비어있을 수 없습니다." }
                require(name.length <= MAX_NAME_LENGTH) { "참가자 이름은 ${MAX_NAME_LENGTH}자 이하여야 합니다." }${MAX_NAME_LENGTH}자 이하여야 합니다.")
```

Reply:
조언 감사합니다! 더 코틀린스럽고 좋은 것 같습니다!